src/Emulator.cpp

```diff
@@ -17,16 +15,13 @@ void Emulator::Start()
 {
     if (Initialize())
     {
-        // Set the current time to the start time
-        Uint32 currentTime = SDL_GetTicks();
+        SDL_Event event;
 
         Logger::Log("Gameboy is up and running!");
-        SDL_Event event;
+
         while (m_isRunning)
         {
             Uint32 frameStartTime = SDL_GetTicks();
```

Comment:
Start time (ticks since the app has started)
End time (ticks since the app started, but after the StepFrame and Render call.

Elapsed time = End time - Start Time!